src/main/java/picard/sam/markduplicates/UmiMetrics.java

```diff
@@ -75,12 +77,17 @@
     /** Estimation of Phred scaled quality scores for UMIs */
     public double UMI_BASE_QUALITIES = 0.0;
 
+    /** The number of UMIs that contain at least one N */
```

Comment:
The percentage of reads that contain a UMI that contains at least one N

Reply:
changed